proxy/src/http/source.rs

```diff
@@ -795,6 +835,7 @@ mod test {
         let want = coco::with_browser(&*peer.lock().await, &urn, |mut browser| {
             coco::blob(
                 &mut browser,
+                None,
```

Comment:
Could use `peer.peed_id` or the peer id derived from the key. Maybe for a later change.

Reply:
I think the heuristic is a bit more complicated and depends on the state of the local monorepo: https://www.notion.so/monadic/Overview-4969e31627984cc1938a6603d7d33738#3f74974beb934d81a911fc1eed1f8a88

Reply:
Here `peer` is `librad::net::peer::PeerApi`.

Reply:
Ya, but we're in tests so passing in `peer.peer_id`, knowing that this `peer` created the projects, seems like a reasonable test to me. But it was just a suggestion :)

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/620/commits/105b9d048bd8d1b6bb8573f496246a6f00b048fb